### PR TITLE
1.19.1

### PR DESCRIPTION
**Pull request by kmuehlbauer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@conda-forge-admin please rerender

**Review by zssherman — APPROVED** [review]

**Comment by kmuehlbauer** [comment]
> * No valid build backend found for Python recipe for package `arm_pyart` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.

@mgrover1 Maybe that's the issue?

**Comment by zssherman** [comment]
@kmuehlbauer extra 'o' in setuptools, I did a quick fix

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,6 +32,7 @@ requirements:
     - pip
     - numpy >=1.18
     - cython >=0.24.0,<3.0.10
+    - setuptoools
```
Comment (zssherman) [inline recipe/meta.yaml]:
```suggestion
    - setuptools
```

**Comment by kmuehlbauer** [comment]
thanks @zssherman for taking care. It's already very late here. If you could have an eye on this and merge it when ready, would be great. Hopefully this will resolve on its own. We miss you :heart:

**Comment by zssherman** [comment]
@kmuehlbauer No worries at all! Hopefully next conference!

**Pull request merged**